src/software/thunderscope/replay/proto_player.py

```diff
@@ -130,6 +134,91 @@ def is_log_entry_corrupt(log_entry: bytes, version: int) -> bool:
         except Exception:
             return True
 
+    def build_chunk_index(self, folder_path: str) -> dict[str:float]:
```

Comment:
Is `dict[str:float]` the correct way to type hint a dictionary? iirc the type args should be comma separated i.e. `dict[str, float]`

Reply:
Good call! My bad, you are right! Will fix.

Reply:
Done